metis-dereference/metis-dereference-service/src/main/java/eu/europeana/metis/dereference/service/MongoDereferencingManagementService.java

```diff
@@ -47,6 +47,21 @@ public void emptyCache() {
     this.processedEntityDao.purgeAll();
   }
 
+  @Override
+  public void purgeByNullOrEmptyXml() {
+    this.processedEntityDao.purgeByNullOrEmptyXml();
```

Comment:
No need for `this` keyword. There is only one class variable instantiated with this name, there is no other object of the same type and name to cause confusion. Remove this